sphinx/themes/agogo/static/rtl.css.jinja

```diff
@@ -0,0 +1,156 @@
+{% if theme_is_rtl|tobool %}
+
+/* Core RTL overrides for Agogo theme */
+body,
+div.header div.rel {
+    direction: rtl !important;
+}
```

Comment:
I’m surprised this is done via CSS rather than the `dir` attribute in HTML. If CSS fails to load, surely the HTML document on its own should be RTL-friendly?